paket.dependencies

```diff
@@ -23,7 +23,7 @@ nuget NUnit.Console
 nuget NUnit3TestAdapter version_in_path: true
 clitool dotnet-mergenupkg
 
-github fsharp/FAKE src/app/FakeLib/Globbing/Globbing.fs
+github fsharp/FAKE:0341a2e614eb2a7f34607cec914eb0ed83ce9add src/app/FakeLib/Globbing/Globbing.fs
```

Comment:
The file moved to `/src/legacy/FakeLib/Globbing/Globbing.fs` that was probably the problem?